src/vcf/validate_optional_policy.cpp

```diff
@@ -146,6 +152,36 @@ namespace ebi
         }
     }
 
+    void ValidateOptionalPolicy::check_body_entry_alt_gvcf_end(ParsingState & state, Record const & record) const
+    {
+        if (std::find(record.alternate_alleles.begin(), record.alternate_alleles.end(), GVCF) != record.alternate_alleles.end()
+            && record.info.find(END) == record.info.end()) {
+            throw new AlternateAllelesBodyError{state.n_lines,
+                    "INFO END should be provided, as " + GVCF + " (symbolic alternate allele) is supposed to represent a region"};
+        }
+    }
+
+    void ValidateOptionalPolicy::check_body_entry_alt_gvcf_gt_value(ParsingState & state, Record const & record) const
+    {
+        if (std::find(record.alternate_alleles.begin(), record.alternate_alleles.end(), GVCF) != record.alternate_alleles.end()
```

Comment:
try to be a bit more strict with the line limit of 120 characters, it really improves readability. in fact, I don't recall talking about this, so this might be the first time you hear about this :sweat_smile: